.gitlab-ci.yml

```diff
@@ -1,40 +1,22 @@
 # B"H
 # This file is a template, and might need editing before it works on your project.
 # Read more about this script on this blog post https://about.gitlab.com/2016/11/30/setting-up-gitlab-ci-for-android-projects/, by Greyson Parrelli
-image: openjdk:8-jdk
-
-variables:
-  ANDROID_COMPILE_SDK: "27"
-  ANDROID_BUILD_TOOLS: "27.0.3"
-  ANDROID_SDK_TOOLS: "26.1.1"
+#image: openjdk:8-jdk
+image: registry.gitlab.com/avimarineinnovatons/android-image:latest
```

Comment:
What is this image? Who created it where is its source control?

Reply:
This is image i'm creating from git-lab Repo in  AviMarineInnovatons account on git-lab
you can check it if you like 
the Android-Image repo contain 

- secret files.
- Dockerfile
- .gitlab-ci.yml

the docker file is based on the auto generated gitlab-ci.yml for android 
with this configuration i got 
- private image saved in a secure repo.
- decreasing the build time significantly because we skip a long installation time of android-sdk

Reply:
I could not find the image file or the yaml which creates.
It should be saved in the git repo anyhow.
EVERYTHING as code

Reply:
Everything is as code exept the image with generated from code in another repo. 

I'll try to write later more details explaination abaut the process.

Reply:
https://gitlab.com/AviMarineInnovatons/android-image.git

this is the other repo 
You must be loged In as our user to be able to see the repo.